app/routes/goal_routes.py

```diff
@@ -1 +1,76 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, Response, make_response, abort
+from ..db import db
+from .route_utilities import validate_model, update_dict_to_database, add_dict_to_database
+from app.models.goal import Goal
+from app.models.task import Task
+
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
```

Comment:
Prefer blueprint to be named bp.
```suggestion
bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
```